docs/docs/hydro/live-collections/streams.md

```diff
@@ -3,10 +3,119 @@ sidebar_position: 2
 ---
 
 # Streams
-You can view the full API documentation for Streams [here](pathname:///rustdoc/hydro_lang/stream/struct.Stream).
+Streams are the most common type of live collection in Hydro; they can be used to model streaming data collections, a feed of API requests, or even time-based intervals. A `Stream` represents a sequence of elements, with new elements being asynchronously appended to the end of the sequence. Streams can be transformed using APIs like `map` and `filter`, based on Rust [iterators](https://doc.rust-lang.org/beta/std/iter/trait.Iterator.html). You can view the full API documentation for Streams [here](pathname:///rustdoc/hydro_lang/stream/struct.Stream).
+
+Streams have several type parameters:
+- `T`: the type of elements in the stream
+- `L`: the location the stream is on (see [Locations](../locations/index.md))
+- `B`: indicates whether the stream is [bounded or unbounded](./bounded-unbounded.md)
+- `Order`: indicates whether the elements in the stream have a deterministic order or not
```

Comment:
Why is `Order` a word and the rest are just a character?

Reply:
I support all type parameters being full words